numpy/typing/tests/test_isfile.py

```diff
@@ -10,7 +10,6 @@
     ROOT / "__init__.pyi",
     ROOT / "char.pyi",
     ROOT / "ctypeslib.pyi",
-    ROOT / "emath.pyi",
```

Comment:
The comment below says `"""Test if all ``.pyi`` files are properly installed."""`. Based on that, the new (and renamed) `.pyi` files should probably be added here?

Reply:
I'd say it'd be fairly redundant here. 

The reason for introducing this test is that numpy has a number of sub-packages (_e.g._ [numpy/lib/setup.py](https://github.com/numpy/numpy/blob/main/numpy/lib/setup.py)), each of which needs to recognize `.pyi` as valid data files. Since the main `numpy/lib/__init__.pyi` file is already tested in aforementioned test, adding these new `.pyi` file adds little value.,